.github/workflows/workflow.yaml

```diff
@@ -150,6 +150,39 @@ jobs:
           path: |
             logs-*.json
 
+  build_mutation_test:
```

Comment:
TOOD for later: add `build_mutation_test` in `needs` for release and pre-release as a release gate